aspnet-core/Diagram/Undo-and-Redo.md

```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
 title: Undo and Redo | Diagram | ASP.NET CORE | Syncfusion
-description: undo and redo
+description: Learn here more about undo and redo support in Syncfusion Essential ASP.NET Core Diagram Control, its elements, and more.
 platform: aspnet-core
 control: Diagram
 documentation: ug
 ---
 
-# History Manager
+# History Manager in ASP.NET Core Diagram
```

Comment:
Change `Undo and Redo` instead of History Manager

Reply:
changed